Update interfaces/web-nfc.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/3157f96/ed/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/633413944

stephenmcgruer reviewed (changes requested):
Partially invalid due to tidoust/reffy#205. Blocking until that is resolved.

autofoolip committed:
Update interfaces/web-nfc.idl

Source: https://github.com/tidoust/reffy-reports/blob/3157f96/ed/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/633413944
